libs/documentation/src/lib/storybook/formly/formly-stepper/formly-stepper-advanced/stepper-advanced.service.ts

```diff
@@ -0,0 +1,392 @@
+import { Injectable } from '@angular/core';
+import { SdsFormlyTypes } from '@gsa-sam/sam-formly';
+import { FormlyFieldConfig } from '@ngx-formly/core';
+
+@Injectable()
+export class StepperAdvancedService {
+  getPermission(): FormlyFieldConfig {
+    return {
+      key: 'permissions',
+      fieldGroupClassName: 'grid-row',
+      fieldGroup: [
+        {
+          key: 'permissionsNeeded',
+          template: '<br/><h4><b>what permission</b></h4></br> Select the specfific <br>',
+        },
+        {
+          key: 'typeOfEntity',
+          className: 'desktop: grid-col-12 tablet: grid-col-12',
+          type: SdsFormlyTypes.RADIO,
+          templateOptions: {
+            label: 'What type of entity are you registering?',
+            hideOptions: true,
+            options: [
+              {
+                label: 'Business Or Organization',
+                value: 'business',
+              },
+              {
+                label: 'U.S. State Government',
+                value: 'stateGovt',
+              },
+              {
+                label: 'U.S. Local Government',
+                value: 'localGovt',
+              },
+              {
+                label: 'Tribal Government',
+                value: 'tribal',
+              },
+              {
+                label: 'Foreign Government',
+                value: 'foreign',
+              },
+            ],
+          },
+        },
+        {
+          key: 'contractOpp',
```

Comment:
Can you check this configuration? The checkboxes are not showing correctly.
<img width="460" alt="image" src="https://github.com/user-attachments/assets/75b0c986-6313-41b0-a3bc-c717de2022c7">

Reply:
I changed 'key' to 'label' as per the formly update and it is now fixed.

Reply:
You've got the human readable text set as the value rather than the label

Reply:
fixed